### PR TITLE
Merging druid rework

### PR DESCRIPTION
**Pull request by Bakelt**
Summon Druid has been reworked to allow a more active playstyle and to allow wolves and grizzly to be more attractive options with non-summoner skills.

Raven
- Reworked to be a more "active" skill for a summoner druid
- Now deals splash damage
- Now has synergy with all druid summon skills
- Cap raised to 10 and hits reduced to 3

Spirit Wolf
- No longer has synergy with Dire Wolf or Grizzly. Previous synergies are now baked into the skill scaling
- Max wolves has been reduced from 10 to 5
- Now has a chance to cast cold twisters on hit
- Now inherits the players cold pierce stat
- Now has a synergy with all cold and wind elemental skills

Dire Wolf
- No longer has synergy with Spirit Wolf or Grizzly. Previous synergies are now baked into the skill scaling
- Max wolves has been reduced from 5 to 3
- Now deals fire damage on hit
- Now has a chance to cast fire storm on hit
- Now inherits the players fire pierce stat
- Now has a synergy with all fire elemental skills and fire claws

Grizzly
- No longer has synergy with Spirit Wolf or Grizzly. Previous synergies are now baked into the skill scaling
- Max bears remains at 2. You can't separate a bonded pair
- Now has a chance to cast shockwave on hit
- Now has a synergy with all physical shapeshift skills

Heart of the Wolverine
- Reworked slightly to be more attractive to all builds.
- %DMG per reduced
- Now gains %IAS and %FCR per hard point

Spirit of the Barbs
- Reworked to be a debuffer spirit
- Now provides an aura that reduces the resistance and physical damage reduction of nearby enemies

Creeper Vine
- Now inherits the players poison pierce stat

Carrion Vine
- Now heals both mana and HP and both have been increased

Solar Creeper
- Now causes physical damage in an area and applies open wounds

### CODE REVIEW

File: data/global/excel/monequip.txt [hunk 1]
```diff
@@ -43,9 +43,7 @@ shadowmaster		0	7ar	rarm	3	7xf	rarm	3	7qr	rarm	3	0
 shadowmaster		0	7cs	larm	3	7tw	larm	3	7qr	larm	3	0
 shadowmaster		0		tors	3							0
 shadowmaster		0		head	3							0
-fenris		0	vf3	tors	7							0
 necroskeleton		0	vf3	tors	7							0
 claygolem		0	vf3	tors	7							0
 bloodgolem		0	vf3	tors	7							0
-druidbear		0	vf3	tors	7							0
```
Comment (CollinHerber) [inline data/global/excel/monequip.txt:50]: This doesn't break anything? Odd. Neat but odd.

**Review by CollinHerber — APPROVED** [review]
Looks all good to me.

**Comment by CollinHerber** [comment]
Wasn't sure if you were ready to merge so I'll leave it here. Next branch is ready for 2.0 merges.

File: data/global/excel/monequip.txt (hunk 1 above)
Comment (Bakelt) [reply data/global/excel/monequip.txt:50]: Monequip.txt basically says "druidbear" gets unique ID "vf3" as a chest piece at lvl1. This is the Summon Armor splash chest

This was replaced with the shockwave on-hit effect in the rework so he doesn't need both.

**Comment by Bakelt** [comment]
> Wasn't sure if you were ready to merge so I'll leave it here. Next branch is ready for 2.0 merges.

It is all good to merge. We may make further tweaks but would prefer to have it merged now so that anyone else working on 2.0 skill changes can work freely without causing major merge conflict.

**Pull request merged**